src/Auth/ControllerAuthorize.php

```diff
@@ -50,7 +50,7 @@ class ControllerAuthorize extends BaseAuthorize
     protected $_Controller;
 
     /**
-     * {@inheritDoc}
+     * @inheritDoc
```

Comment:
Do you know if api-gen will also follow this style of tag?

Reply:
I do not. We should find out. However, it should, since this is the correct tag for full overwrite.

Reply:
We will have to make updates/fixes to apigen anyway to use with 4.x, regardless of this change :slightly_smiling_face:.